src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -558,13 +556,15 @@ void sizeZoneSpaceEquipmentPart1(EnergyPlusData &state,
     } else {
         nonAirSystemResponse = SysOutputProvided;
         if (state.dataHeatBal->doSpaceHeatBalance) {
-            for (int spaceNum : state.dataHeatBal->Zone(zoneNum).spaceIndexes) {
+            for (int spaceNumLoop : state.dataHeatBal->Zone(zoneNum).spaceIndexes) {
```

Comment:
[src/EnergyPlus/ZoneEquipmentManager.cc:561]:(style),[shadowArgument],Local variable 'spaceNum' shadows outer argument

Reply:
This is the other thing causing a conflict.  Obviously this is minor, and it looks like you also changed this to be spaceNum2 or something.

Reply:
Yes, you can keep either one `spaceNum2` or `spaceNumLoop`. I'll look at the other conflict.